packages/core/src/Stores/Constants/markers.js

```diff
@@ -1,7 +1,7 @@
 import { localize } from '@deriv/translations';
-import MarkerLine from '../../Components/markers/marker-line';
-import MarkerSpotLabel from '../../Components/markers/marker-spot-label';
-import MarkerSpot from '../../Components/markers/marker-spot';
+import MarkerLine from '../../Components/markers/marker-line.jsx';
```

Comment:
Can not use the alias for the path?

Reply:
added Components to webpack aliases and changed it here as well.

Reply:
I think It would be better to handle this in a separate PR. other paths other than Components are added to the webpack alias So I think there are a lot of files that needs change.
what do you think?